src/ui_contextmenu.cpp

```diff
@@ -31,6 +32,9 @@ class OpenLyricsContextSubItem : public contextmenu_item_simple
             case cmd_manualsearch_lyrics: out = "Search for lyrics (manually)"; break;
             case cmd_edit_lyrics: out = "Edit lyrics"; break;
             case cmd_mark_instrumental: out = "Mark as instrumental"; break;
+            case cmd_seek_to_next_lyric: out = "Seek to next lyric timestamp"; break;
+            case cmd_seek_to_prev_lyric: out = "Seek to prev lyric timestamp"; break;

```

Comment:
We shouldn't be using code-like abbreviations on the UI (same applies to the description down below).

```suggestion
            case cmd_seek_to_prev_lyric: out = "Seek to previous lyric timestamp"; break;
```